ANSWERS.md

```diff
@@ -1,4 +1,17 @@
 - [ ] Why would you use class component over function components (removing hooks from the question)?
+
+    Because in the past, it solved many problems, and on a vary large scale. So it may be more cost-effective to continue writing with class components rather than spend man-hours and money on converting to functional components
+
```

Comment:
This is probably just because how the questions was worded, but if there were no hooks why would you use a class component over a functional component?